src/Network/Network/Models/PSPrivateLinkServiceConnectionState.cs

```diff
@@ -36,6 +36,6 @@ public partial class PSPrivateLinkServiceConnectionState
         public string Status { get; set; }
         public string Description { get; set; }
         [Ps1Xml(Target = ViewControl.Table)]
-        public string ActionRequired { get; set; }
+        public string ActionsRequired { get; set; }
```

Comment:
This could be considered breaking change. Please sync up with PS team. Alternative would be to add `.AfterMap` in mapping between ConnectionState's PS and SDK models in `NetworkResourceManagerProfile.cs`

Reply:
We don't have customer using it yet. So, in effect, it's not considered as a breaking change, long as we release it ASAP.

Reply:
Even if no customer using it, it is still breaking change anyway.
Please follow the below steps add breaking change attribute then it can be released in the next breaking change release next year.